plugins/metrics/sync.go

```diff
@@ -7,7 +7,7 @@ import (
 var isTangleTimeSynced atomic.Bool
 
 func measureSynced() {
-	tts := deps.Tangle.TimeManager.Synced()
```

Comment:
we could add the synced field to the metrics here as well so it's visible in Grafana dashboard. But this can be done in a separate PR which also updates the dashboard.

Reply:
Done